File: AppGenerateFiles/administrative_access/Model/Transactional_Configuraciones.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transactional_Configuraciones : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Configuracion { get; set; }
       public string? Nombre { get; set; }
       public string? Descripcion { get; set; }
       public string? Valor { get; set; }
       public string? Tipo_Configuracion { get; set; }
   }
}

File: AppGenerateFiles/administrative_access/FrontModel/Transactional_Configuraciones.js
//@ts-check
import { EntityClass } from "../../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../../WDevCore/WModules/CommonModel.js";
class Transactional_Configuraciones extends EntityClass {
   /** @param {Partial<Transactional_Configuraciones>} [props] */
   constructor(props) {
       super(props, 'EntityAdministrative_access');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Configuracion;
   /**@type {String}*/ Nombre;
   /**@type {String}*/ Descripcion;
   /**@type {String}*/ Valor;
   /**@type {String}*/ Tipo_Configuracion;
}
export { Transactional_Configuraciones }
